blueocean-core-js/src/js/config.js

```diff
@@ -7,6 +7,12 @@ const config = blueocean.config || {};
 const features = config.features || {};
 const organization = blueocean.organization || {};
 
+// any all features added by ?features=SOMETHING,SOMETHING_ELSE
+const pfx = 'features=';
+const pfxlen = pfx.length;
+(window.location.href.split('?')[1] || '').split('&')
```

Comment:
@kzantow that introduced a regression bug for unit testing...you are not catching a possible NPE!

`window.location` in unit testing is normally `undefined` and can be until `href` 

:bug: